popup.js

```diff
@@ -12,12 +12,31 @@ function sendRequest(data) {
 }
 
 jQuery(document).ready(function($) {
-    //send message request to content script
-    $('#btnUpload').click(function() {
-        sendRequest({
-            type: 'addJson',
-            jsontext: $('#jsontext').val()
-        });
+    $('#form-import').submit(function(e) {
```

Comment:
@jinhduong I change from button's event to form submit event because when I see your code at this line: https://github.com/jinhduong/highlighter-basic/blob/master/popup.html#L14

You used <form> tag, but all other tag is outside <form>. So I brought them to where they should belong (with submit event instead of click event).